### PR TITLE
Migrate Create|Commit to ginkgo

### PR DESCRIPTION
**Pull request by baude**
Migrate create and commit bats tests to the ginkgo
test suite.  In doing so, some structures had to be
moved to pkg/podmanstructs/podmanstructs.go so we
could do better verification of test results.

Signed-off-by: baude <bbaude@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
👍

File: pkg/podmanstructs/podmanstructs.go [hunk 1]
```diff
@@ -0,0 +1,204 @@
+package podmanstructs
```
Comment (mheon) [inline pkg/podmanstructs/podmanstructs.go]: Could you rename this to inspectstructs? Or inspecttypes?

Comment (mheon) [reply pkg/podmanstructs/podmanstructs.go]: Maybe even plain old "inspect" to keep things simple

Comment (rhatdan) [reply pkg/podmanstructs/podmanstructs.go]: Yes I like that.

Comment (baude) [reply pkg/podmanstructs/podmanstructs.go]: fixed

**Comment by mheon** [comment]
LGTM pending tests

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Pull request closed without merging**